src/Microsoft.Health.Fhir.Core/Features/Conformance/CapabilityStatementBuilder.cs

```diff
@@ -99,7 +99,7 @@ public ICapabilityStatementBuilder ApplyToResource(string resourceType, Action<L
 
             if (resourceComponent == null)
             {
-                resourceComponent = new ListedResourceComponent(_configuration)
+                resourceComponent = new ListedResourceComponent(_configuration, resourceType)
```

Comment:
I'm wondering if it is now easier to just make `DefaultOptionHashSet.Default` have a setter so this logic can be done below in the builder

Reply:
I can take a look and see what can be done.

Reply:
@brendankowitz I made the change. Let me know if the changes correlate to what you were asking.